### ISSUE TITLE
Two bugs that I found on a server with SSC enabled

### ISSUE THREAD

**Issue by Skiret** [Pryaxis/TShock#2904]
* TShock version: 5.1.3

#### Reproduction steps (if applicable)?

Mistake #1
1. I put on "Brain of Confusion" and when dodging a monster, all buffs and debuffs disappear.

Mistake #2
1. I wear one of the "Titan Glove", "Power Glove", "Mechanical Glove" or "Fire Gauntlet"
2. When using "The Horsemans Blade" and stronger swords, the character is blocked by a spider web.

**Comment by sgkoishi** [comment]
Duplicate of #2799 and #2804 . Fixed for next release, or download the experiment version.
